### PR TITLE
[PREVIEW] update to azure vault

### PR DESCRIPTION
**Pull request by david-gimelle**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by borislavpetrovhmcts** [comment]
You need to rebase this with master and test again

File: infrastructure/main.tf [hunk 1]
```diff
@@ -88,14 +88,22 @@ provider "vault" {
   address = "https://vault.reform.hmcts.net:6200"
 }
```
Comment (mthava) [inline infrastructure/main.tf]: remove this you dont need this anymore (entire block)

Comment (david-gimelle) [reply infrastructure/main.tf]: Done

File: infrastructure/main.tf [hunk 2]
```diff
@@ -84,18 +84,22 @@ module "frontend" {
   }
 }
 
-provider "vault" {
```
Comment (tchow8) [inline infrastructure/main.tf]: Please add this back in. Actually required, sorry.

Comment (mthava) [reply infrastructure/main.tf]: this required because the state file contains some reference to the old key. The question is how we are going to fix this

Comment (mthava) [reply infrastructure/main.tf]: We can either fix the state file or keep these as a short term solution

Comment (tchow8) [reply infrastructure/main.tf]: Cool, ignore this change request please. I'll clear the state files. Cheers

**Comment by nsingamsetty** [comment]
Test is completed and compared both AAT and preview environment and the keys are not changed against AAT. the only change i see is below keys and it is acceptable after discussion with Tony.

APPLICATION_INSIGHTS_IKEY and APPINSIGHTS_INSTRUMENTATIONKEY are fine and acceptable.

**Pull request merged**